docs/BasicCommunication/GetAppProperties/index.md

```diff
@@ -0,0 +1,93 @@
+## GetAppProperties
+
+Type
+: Function
+
+Sender
+: HMI
+
+Purpose
+: Get the current properties of an application
+
+
+### Request
+
+#### Parameters
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|policyAppID|String|false|maxlength: "100"|
+
+
+### Response
+In case `policyAppID` is omitted in the request, SDL provides all app properties in the response.
+
+#### Parameters
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|properties|[Common.AppProperties](../../common/structs/#appproperties)|false|array: true<br>minsize: 1|
+
+
+### Example Request
+```json
+{
+  "id" : 47,
+  "jsonrpc" : "2.0",
+  "method" : "BasicCommunication.GetAppProperties",
+  "params" :
+  {
+    "policyAppID" : 123456
+  }
+}
+```
+
+### Example Response
+
+```json
```

Comment:
This JSON is still formatted weirdly.
Both `Example Request` and `Example Error` in this file look fine.
Since both other examples use a space before and after the `:`, please do the same for this example.

The biggest issue is that closing brackets should be at the same indentation level as their opening tag.